src/main/java/com/zufar/onlinestore/cart/dto/AddNewProductToShoppingSessionRequest.java

```diff
@@ -2,7 +2,12 @@
 
 import jakarta.validation.constraints.NotNull;
 
-public record AddNewItemToShoppingSessionRequest(
+import java.util.UUID;
+
+public record AddNewProductToShoppingSessionRequest(
+
+        @NotNull(message = "UserId is the mandatory attribute")
+        UUID userId,
```

Comment:
I can be wrong, but in my opinion we do not need this attribute

Reply:
Fixed